Cargo.toml

```diff
@@ -61,3 +61,4 @@ tonic-build = {version = "0.9.2",features = ["prost"]}
 members = [
     "dialdbg",
 ]
+exclude = [ "examples" ]
```

Comment:
(flyby) examples thought of themselves as part of the workspace, need to exclude them or else they won't run.